CONTRIBUTING/index.html

```diff
@@ -1,4182 +1,63 @@
-<h3 id="committing">Committing<a class="headerlink" href="#committing" title="Permanent link">&para;</a></h3>
-<p>See the <a href="../running-locally/#committing">Committing Guidelines</a>.</p>
-<h3 id="dependencies">Dependencies<a class="headerlink" href="#dependencies" title="Permanent link">&para;</a></h3>
-<p>Dependencies increase the risk of security issues and have on-going maintenance costs.</p>
-<p>The dependency must pass these test:</p>
-<ul>
-<li>A strong use case.</li>
-<li>It has an acceptable license (e.g. MIT).</li>
-<li>It is actively maintained.</li>
-<li>It has no security issues.</li>
-</ul>
-<p>Example, should we add <code>fasttemplate</code>
-, <a href="https://snyk.io/advisor/golang/github.com/valyala/fasttemplate">view the Snyk report</a>:</p>
-<table>
-<thead>
-<tr>
-<th>Test</th>
-<th>Outcome</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td>A strong use case.</td>
-<td>❌ Fail. We can use <code>text/template</code>.</td>
-</tr>
-<tr>
-<td>It has an acceptable license (e.g. MIT)</td>
-<td>✅ Pass. MIT license.</td>
-</tr>
-<tr>
-<td>It is actively maintained.</td>
-<td>❌ Fail. Project is inactive.</td>
-</tr>
-<tr>
-<td>It has no security issues.</td>
-<td>✅ Pass. No known security issues.</td>
-</tr>
-</tbody>
-</table>
-<p>No, we should not add that dependency.</p>
-<h3 id="test-policy">Test Policy<a class="headerlink" href="#test-policy" title="Permanent link">&para;</a></h3>
-<p>Changes without either unit or e2e tests are unlikely to be accepted.
-See <a href="https://github.com/argoproj/argo-workflows/blob/main/.github/pull_request_template.md">the pull request template</a>.</p>
-<h3 id="contributor-workshop">Contributor Workshop<a class="headerlink" href="#contributor-workshop" title="Permanent link">&para;</a></h3>
-<p>Please check out the following resources if you are interested in contributing:</p>
-<ul>
-<li><a href="https://youtu.be/zZv0lNCDG9w">90m hands-on contributor workshop</a>.</li>
-<li><a href="https://docs.google.com/presentation/d/1IU0a3unnr3tBRi38Zn3EHQZj3z6yvocfG9x9icRu1LE/edit?usp=sharing">Deep-dive into components and hands-on experiments</a>.</li>
-<li><a href="https://github.com/argoproj/argo-workflows/blob/main/docs/architecture.md">Architecture overview</a>.</li>
-</ul>
-
-              
-
-<!-- Giscus -->
-<h2 id="__comments">Comments</h2>
-
-<script src="https://giscus.app/client.js"
-        data-repo="argoproj/argo-workflows"
-        data-repo-id="MDEwOlJlcG9zaXRvcnkxMDA5ODI0NDk="
-        data-category="General"
-        data-category-id="MDE4OkRpc2N1c3Npb25DYXRlZ29yeTMyMTA3OTY1"
-        data-mapping="pathname"
-        data-reactions-enabled="1"
-        data-emit-metadata="0"
-        data-input-position="bottom"
-        data-theme="light"
-        data-lang="en"
-        data-loading="lazy"
-        crossorigin="anonymous"
-        async>
-</script>
-
-
+              <h1>Contributing - Argo Workflows - The workflow engine for Kubernetes</h1>
```

Comment:
I didn't have a chance to review this follow-up PR, but this seems to have used an older version of the script? As this should just be "Contributing"

This seems to be the case for all pages currently live, which is how I noticed it (clicking on an old link). i.e. `heading == title` in this PR when it shouldn't

Reply:
Looking at the script, it looks to have the erroneous code currently. The [revision](https://gist.github.com/jmeridth/94cd15d2b11d444fdf91913aa59f34ab/revisions) 2 diffs ago (GH doesn't seem to have a direct link) has this diff, which should be reverted:

```diff
-                page_h1 = line.replace(OPEN_TITLE_HTML, '').replace(CLOSE_TITLE_HTML, '').replace(f" - {BASE_TITLE}", '').strip()
+                page_h1 = line.replace(OPEN_TITLE_HTML, '').replace(CLOSE_TITLE_HTML, '').strip()
```

Reply:
Fixed in #12500